servant-client/test/Servant/ClientTestUtils.hs

```diff
@@ -74,6 +82,22 @@ instance FromForm Person
 instance Arbitrary Person where
   arbitrary = Person <$> arbitrary <*> arbitrary
 
+instance MimeRender PlainText Person where
+  mimeRender _ = LazyByteString.fromStrict . encodeUtf8 . Text.pack . show
```

Comment:
I guess you could use `Data.ByteString.Lazy.Char8.pack . show` here.

Reply:
But that would work only for ASCII, not UTF8, right?

I would prefer to leave it as is, because it's more obvious that it works, and because I think it's not important.  Let me know if I'm missing anything.

Reply:
Hm, you are right, sorry!